resources/prosody-plugins/mod_visitors.lua

```diff
@@ -246,8 +255,11 @@ process_host_module(main_muc_component_config, function(host_module, host)
         -- a message from visitor occupant of known visitor node
         stanza.attr.from = to;
         for _, o in room:each_occupant() do
-            -- send it to the nick to be able to route it to the room (ljm multiple rooms) from unknown occupant
-            room:route_to_occupant(o, stanza);
+            -- Ignore sending messages to configured domains (jibri and transcribers)
+            if not ignore_list:contains(jid.host(o.bare_jid)) then
```

Comment:
Why filter messages *to* jibri? I don't know if jibri currently displays chat messages in any way, but I think it should be able to.

Reply:
Hah ok, I will drop that.